app/app.go

```diff
@@ -16,7 +16,7 @@ type App struct {
 	wp     *WrappedPolicy
 	vc     *resources.VaultClient
 	router *gin.Engine
-	paused *bool
+	isRunning *bool
```

Comment:
Eh the formatting is off, think you did a search and replace here without explicitly doing a manual save.

Anyway, nitpicking: for types costing less than a 32-bit/64-bit (such as `isRunning`, which is a 1 byte `bool`), should just use the value type itself without the pointer to reduce performance penalty. https://medium.com/@vCabbage/go-are-pointers-a-performance-optimization-a95840d3ef85.

No issue even if you need a reference / pointer to `isRunning` to mutate it in some other function, because you can either just pass a pointer of `App`, or pointer of `isRunning`.